carbon-design-system/carbon
# fix(Checkbox): add positioning to checkbox input for native validation alignment

Closes #10002


#### Changelog

- even thought checkbox input isn't visible, this aligns it to the visible checkbox so that native browser validation will be aligned
- add validation example on checkbox stories (can remove this once approved)

#### Testing / Reviewing

View React deploy preview:
1. go to checkbox validation story
2. don't check the box
3. click the button
4. browser validation should align below checkbox
